src/explore-education-statistics-frontend/src/modules/table-tool/components/FiltersForm.tsx

```diff
@@ -75,6 +83,21 @@ const FiltersForm = (props: Props & InjectedWizardProps) => {
           ),
         ),
       })}
+      preSubmit={values => {
+        const newValues = values;
+        // Automatically select totalValue for filters that haven't had a selection made
+        Object.keys(values.filters).forEach(filterName => {
+          if (
+            newValues.filters[filterName].length === 0 &&
+            subjectMeta.filters[filterName].totalValue
+          ) {
+            newValues.filters[filterName].push(
```

Comment:
Definitely shouldn't be using mutation like this. Mutation is particularly bad in React as there aren't really any guarantees that it will update component state correctly.

Instead we should be using Formik's `setFieldValue` (see [here](https://jaredpalmer.com/formik/docs/api/formik#setfieldvalue-field-string-value-any-shouldvalidate-boolean-void)). Alternatively you could even call the `onSubmit` handler manually with the correct form values on the submit button's `onClick` (and `preventDefault` the submit event).